### DISCUSSION TITLE
Expand Search Result to show Distance , Average Speed, and Total Climb

### DISCUSSION THREAD

**Discussion by stvzldn** [mytourbook/mytourbook#1378] [Ideas]
Often I will do a Tour Search to compare a current tour with prior.  However the Result display options are limited to Description, Start, and Weather plus the start date/time.  I have to open each tour to see speed and elevation gain to compare to my latest effort.
Would be nice to have the results also sorted by speed or elevation as well as by date,

**Comment by wolfgang-ch** [comment]
Can you create a feature request for it

**Comment by wolfgang-ch** [comment]
Feature request https://github.com/mytourbook/mytourbook/issues/1379
